CONTRIBUTING.md

```diff
@@ -87,7 +87,6 @@ This Wiki is intended to be for _anyone_ -- regardless of expertise -- who is in
 
 - Fork this repository in Github's web interface by pressing the fork icon.
 - Clone your fork and `cd` into the new directory: `git clone https://github.com/username/wiki && cd wiki`
```

Comment:
- Clone your fork to your local machine. and ....

Reply:
they may not be on there local machine i think ill leave as is for now